scripts/monitoring/ops-metric-client.py

```diff
@@ -64,6 +65,21 @@ def run(self):
 
         self.metric_sender.send_metrics()
 
+    @staticmethod
+    def adjust_type(val, metric_type):
+        """
+        :param val: string representing metric value
+        :param metric_type: string or numeric
+        :return: value converted to float if metric_type isn't string
+        """
+        if metric_type == 'string':
+            return val
+
+        try:
+            return json.loads(val)
```

Comment:
LGTM.  Ideally ValueError would be an error, requiring either quotes or type 'string' for strings, but if you can't due to backwards compatibility, this is as good as it gets.

P.S. will you support other `metric_type`s?  E.g. 'float' with val == '1', should it parse as 1.0 or 1?